src/main/java/com/example/dog/dao/JdbcDogDAO.java

```diff
@@ -26,12 +27,16 @@ public JdbcDogDAO(DriverManagerDataSource dataSource) {
         this.dataSource = dataSource;
     }
 
+    @Transactional(readOnly = true)
```

Comment:
That's too early - this annotation won't work for now.

Reply:
oops, forgot tp delete it